x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/step/info/AllocationInfo.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.core.ilm.step.info;
+
+import org.elasticsearch.common.ParseField;
+import org.elasticsearch.common.Strings;
+import org.elasticsearch.common.xcontent.ConstructingObjectParser;
+import org.elasticsearch.common.xcontent.ToXContentObject;
+import org.elasticsearch.common.xcontent.XContentBuilder;
+
+import java.io.IOException;
+import java.util.Objects;
+
+public class AllocationInfo implements ToXContentObject {
+    private final long actualReplicas;
+    private final long numberShardsLeftToAllocate;
+    private final boolean allShardsActive;
+    private final String message;
+
+    static final ParseField ACTUAL_REPLICAS = new ParseField("actual_replicas");
```

Comment:
I think "actual_replicas" is a little difficult to tell what it means, maybe there's a clearer name we could use?

Reply:
Yes, I agree. It'll be a breaking change, but possibly going with the well known `number_of_replicas` would be better?